system/modules/core/library/Contao/File.php

```diff
@@ -280,16 +280,6 @@ public function __get($strKey)
 								\Image::getPixelValue($svgElement->getAttribute('height'))
 							);
 						}
-						elseif ($svgElement->getAttribute('viewBox'))
```

Comment:
Why did you remove the else condition here? If there is no `width` and `height` attribute, there might still be a `viewBox` attribute.